pushapkscript/googleplay.py

```diff
@@ -7,7 +7,7 @@
 
 log = logging.getLogger(__name__)
 
-_AUTHORIZED_PRODUCTS_TO_REACH_GOOGLE_PLAY = ('aurora', 'beta', 'release', 'focus')
+_AUTHORIZED_PRODUCTS_TO_REACH_GOOGLE_PLAY = ('aurora', 'beta', 'release', 'focus', 'reference-browser')
```

Comment:
I would keep that one out, until we're ready to upload the first reference-browser APK. What do you think?

Reply:
I've reverted `do_not_contact_google_play` from the other PR from now, I don't want to throw too many changes into the mix.
I'll remove `reference-browser` from the list of authorized products for now :)